tools/StaticAnalysis/Exceptions/BreakingChangeIssues.csv

```diff
@@ -787,4 +787,45 @@
 "d:\workspace\powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.RedisCache\Microsoft.Azure.Commands.RedisCache.dll","Microsoft.Azure.Commands.RedisCache.SetAzureRedisCache","Set-AzureRmRedisCache","0","1050","The parameter set 'OnlyMaxMemoryPolicy' for cmdlet 'Set-AzureRmRedisCache' has been removed.","Add parameter set 'OnlyMaxMemoryPolicy' back to cmdlet 'Set-AzureRmRedisCache'."
 "d:\workspace\powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.RedisCache\Microsoft.Azure.Commands.RedisCache.dll","Microsoft.Azure.Commands.RedisCache.GetAzureRedisCache","Get-AzureRmRedisCache","0","1050","The parameter set 'GetByResourceGroup' for cmdlet 'Get-AzureRmRedisCache' has been removed.","Add parameter set 'GetByResourceGroup' back to cmdlet 'Get-AzureRmRedisCache'."
 "d:\workspace\powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.RedisCache\Microsoft.Azure.Commands.RedisCache.dll","Microsoft.Azure.Commands.RedisCache.GetAzureRedisCache","Get-AzureRmRedisCache","0","1050","The parameter set 'GetByRedisCache' for cmdlet 'Get-AzureRmRedisCache' has been removed.","Add parameter set 'GetByRedisCache' back to cmdlet 'Get-AzureRmRedisCache'."
-"d:\workspace\powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.RedisCache\Microsoft.Azure.Commands.RedisCache.dll","Microsoft.Azure.Commands.RedisCache.GetAzureRedisCache","Get-AzureRmRedisCache","0","1050","The parameter set 'GetAllInSubscription' for cmdlet 'Get-AzureRmRedisCache' has been removed.","Add parameter set 'GetAllInSubscription' back to cmdlet 'Get-AzureRmRedisCache'."
\ No newline at end of file
+"d:\workspace\powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.RedisCache\Microsoft.Azure.Commands.RedisCache.dll","Microsoft.Azure.Commands.RedisCache.GetAzureRedisCache","Get-AzureRmRedisCache","0","1050","The parameter set 'GetAllInSubscription' for cmdlet 'Get-AzureRmRedisCache' has been removed.","Add parameter set 'GetAllInSubscription' back to cmdlet 'Get-AzureRmRedisCache'."
```

Comment:
Can you remove all of these, and only add the ones causing the build to fail? Many of these don't apply anymore I believe (I'll tag an example).

Reply:
Resolved.